vuu-ui/packages/vuu-layout/src/layout-persistence/LayoutPersistenceManager.ts

```diff
@@ -35,12 +35,26 @@ export interface LayoutPersistenceManager {
    *
    * @returns Full JSON representation of the layout corresponding to the provided ID
    */
-  loadLayout: (id: string) => LayoutJSON;
+  loadLayout: (id: string) => Promise<LayoutJSON>;
 
   /**
    * Retrieves metadata for all existing layouts
    *
    * @returns an array of all persisted layout metadata
    */
-  loadMetadata: () => LayoutMetadata[];
+  loadMetadata: () => Promise<LayoutMetadata[]>;
+
+  /**
+   * Retrieves the current layout
+   *
+   * @returns Full JSON representation of a layouts
+   */
+  loadCurrentLayout: () => Promise<LayoutJSON>;
+
+  /**
+  * Saves the current layout
+  *   
+  * @param layout - Full JSON representation of layout to be saved
+  */
+   saveCurrentLayout: (layout: LayoutJSON) => Promise<void>
```

Comment:
I agree this can be confusing. I think the confusion is originating from the fact that we are using the word "layout" for 2 different things. The layout that we are saving and getting by id is the content of a tab which can be composed of different tables (yellow in the image). The layout from currentLayout is everything inside the "layoutView" which can contain multiple tabs (red in the image)
<img width="795" alt="1" src="https://github.com/ScottLogic/finos-vuu/assets/98337074/81e6b28b-0a4b-4154-a0b2-76fc45481ab7">
we should probably come up with better naming for the 2 so to remove the ambiguity